README.rst

```diff
@@ -66,3 +66,40 @@ To see full usage instructions for the ``start`` action, use ``-h``/``--help``:
       --secondary-nodes node [node ...]
                             Nodes of the secondary-nodes group (default:
                             ['node-2'])
+
+To see full usage instructions for the ``build`` action, use ``-h``/``--help``:
+
+.. code-block:: console
+
+    $ clusterdock build topology_hdp -h
```

Comment:
Nit: might be worth updating this console output since some changes happened since the framework reviews.